.github/workflows/docker.yml

```diff
@@ -0,0 +1,58 @@
+name: Create and publish Docker images
+
+on:
+  push: { branches: [ unstable ] }
+  pull_request: { branches: [ unstable ] }
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - name: github_ci
+        #- name: jenkins_ci # flatironinstitute/cthyb:master-ubuntu-clang not accessible
+        - name: mpich
+        - name: openmpi
+        #- name: triqs_default # flatironinstitute/triqs:unstable-ubuntu-clang not accessible
```

Comment:
the jenkins_ci and triqs_default can be removed. The triqs default dockerfile can be removed completely I think and jenkins_ci is only relevant for jenkins, with docker images only available locally I think?

Reply:
Should I then delete `triqs_default_Dockerfile`?

Reply:
yes it can go. Thanks.